examples/ocp_logging_tests_private_e2e_vars.yaml

```diff
@@ -0,0 +1,9 @@
+---
+
+ocp_logging_tests_private_e2e_enabled: false #Set to true to enable logging e2e tests from openshift-tests-private repo
```

Comment:
This variable is not being used. Please update this file [main.yml](https://github.com/ocp-power-automation/ocp4-playbooks-extras/blob/main/playbooks/main.yml).

Reply:
Imported the role in playbooks/main.yml.